### ISSUE TITLE
Add boolean to UpdateInterceptor to prevent MethodWriter from writing

### ISSUE THREAD

**Issue by JerryShea** [OpenHFT/Chronicle-Wire#185]
See #170

**Comment by RobAustin** [comment]
see @peter-k-lawrey comments below:

The  MethodWriterInterceptors had many conflated concerns which were poorly tested/documented.

I would favour breaking out this functionality to a switch call you can make to turn output on and off.
I think this would make it's purpose clearer and still be easy to implement.

e.g.
````
interface MethodWriter {
    void marshallableOut(MarshallableOut out);
    // optionally
    void closeable(Closeable closeable);
}
````
During replay, it would be set to a /dev/null implementation.

**Comment by RobAustin** [comment]
@JerryShea you may want to revert this change and use a NULL implementation instead, as peter suggests in the comment above.

**Comment by hft-team-city** [comment]
Released in [Chronicle-Wire-2.20.6](https://github.com/OpenHFT/Chronicle-Wire/releases/chronicle-wire-2.20.6), [BOM-2.20.22](https://github.com/OpenHFT/OpenHFT/releases/chronicle-bom-2.20.22)

**Comment by hft-team-city** [comment]
Released in [Chronicle-Wire-2.20.101](https://github.com/OpenHFT/Chronicle-Wire/releases/chronicle-wire-2.20.101), [BOM-2.20.134](https://github.com/OpenHFT/OpenHFT/releases/chronicle-bom-2.20.134)
